packages/connect-wallet/src/hooks/useMiddleware.ts

```diff
@@ -145,8 +161,21 @@ export const useMiddleware = ({
           fetchEns({address: wallet.address, chain: {...rest}, provider}),
         );
       }
+      // If enableDelegateCash is true, we will fetch the delegations
+      // and the fetchDelegations.fulfilled listener will wait until
+      // the action is fulfilled to attribute the order
+      //
+      // If enableDelegateCash is false, we will not fetch the delegations
+      // and attribute the order immediately
```

Comment:
nit: for large comment blocks like this can we make use of block comments instead of multiple single-line comments?

```suggestion
      /**
       * If enableDelegateCash is true, we will fetch the delegations
       * and the fetchDelegations.fulfilled listener will wait until
       * the action is fulfilled to attribute the order
       *
       * If enableDelegateCash is false, we will not fetch the delegations
       * and attribute the order immediately
       */
```

Reply:
^ cc @QuintonC 

My guess is because it's using the block comment for it's designed purpose of multi-line comments. However I have seen multiple single line comments used fairly often so I don't think it's necessarily a bad practice. I'm in favour of consistency with the rest of the codebase / Shopfiy.